request-interfaces.go

```diff
@@ -136,6 +136,13 @@ type NameLookupFileLister interface {
 // error if at end of list. This is testable by comparing how many you
 // copied to how many could be copied (eg. n < len(ls) below).
 // The copy() builtin is best for the copying.
+// ListerAt provides entries that satisfy the os.FileInfo interface.
+// Uid and gid information will on unix systems be retrieved from
+// [os.FileInfo.Sys] if this function returns a [syscall.Stat_t].
+// Alternatively, if the entry implements [FileInfoUidGid], it will be
+// used for uid and gid information.
+// If the entry implements [FileInfoExtendedData], extended attributes will
+// also be returned to the client.
```

Comment:
One last nitpick: paragraphs. It seems it was not well paragraphed already, but we can take this opportunity to clean up the text.

Also, if we could use semantic line breaks: https://sembr.org/ over arbitrarily wrapping at around 80 cols.

Broadly:
```
// InterfaceName does an important thing presented "as a headline".
//
// Important details about the InterfaceName that are tricky or notable in the implementation,
// but not so broadly applicable that they need to be in the headline.
//
// If a Special Case applies,
// then we note the details about that special case.
// It should start with the context of when the condition applies,
// so readers can skip the whole paragraph if the condition is irrelevant to them.
//
// If more Special Cases apply,
// then they should keep following on new paragraphs.
// Putting each Special Case in its own paragraph means readers can quickly skip to the next paragraph.
//
// If a set of special conditions exist,
// and each is fairly straight-forward enough to just list them, go ahead and do that directly:
//
//	A function domain = a special cases
//	A type name: and it’s short simple situation
//
// Note: Notes should show up at the bottom,
// so that readers can skim up from the end for quick access to gotchas, and other minutiae.
```